web/src/js/flow/utils.js

```diff
@@ -49,7 +49,14 @@ export var MessageUtils = {
         } else if (message === flow.response) {
             message = "response";
         }
-        return `/flows/${flow.id}/${message}/content` +  (view ? `/${view}` : '');
+        if (global.MITMWEB_STATIC) {
```

Comment:
Yeah, we could do that and have `.json` in all requested url. 
AFAIK, `GET` operation on a `.json` url is natural, but `POST` or `PUT` operation on a `.json` url sounds so wired. And I'm not sure whether it meets the guideline of RESTful API design...